utils/LLM_summary_shortner.py

```
@@ -0,0 +1,158 @@
+        new_summary (str): The new summary generated by the API.
+
+    Raises:
+        ValueError: If the API key is not found in the environment.
+        requests.HTTPError: For any HTTP errors that occur during the API call.
+    """
+
+
+    # Retrieve API key from environment variable
+    logging.info(
+        f"Initiating API call with the provided input: {current_summary}")
+    api_key = os.environ.get("x_magicapi_key")
```

Ruff (0.8.2)

Use capitalized environment variable `X_MAGICAPI_KEY` instead of `x_magicapi_key`

(SIM112)